Bottom: b8f0efe4fae6507de6744fc69fc65725ce922432
Top:    632823a538bbfdade36462b45ceb8c0ebe081d6b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-20 23:25:06 +0100

Refresh of gl-use-a-vbo-to-avoid-data-cop

---

diff --git a/src/hid/common/hidgl.h b/src/hid/common/hidgl.h
index 8fcf26e..0598b9c 100644
--- a/src/hid/common/hidgl.h
+++ b/src/hid/common/hidgl.h
@@ -24,7 +24,7 @@
 #define __HIDGL_INCLUDED__
 
 #define TRIANGLE_ARRAY_SIZE 5461
-#define TRIANGLE_ARRAY_BYTES (3 * 3 * TRIANGLE_ARRAY_SIZE)
+#define TRIANGLE_ARRAY_BYTES (3 * 3 * sizeof (GLfloat) * TRIANGLE_ARRAY_SIZE)
 typedef struct {
 //  GLfloat triangle_array [3 * 3 * TRIANGLE_ARRAY_SIZE];
   GLfloat *triangle_array;
